sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -150,6 +149,7 @@ trait Interpreter extends ScorexLogging {
     }
   }
 
+  /** Reduces `exp` to SigmaProp under the default (empty) environment. */
```

Comment:
This comment does not sat anything beyond what can be trivially deduced from the code (even with no language knowledge probably), is it needed then ?

Reply:
I think no harm to be explicit. Updated comment.